twake/backend/node/package.json

```diff
@@ -9,7 +9,7 @@
     "build:copy-assets": "npm run build:copy-config",
     "build:copy-config": "cp -R config dist",
     "dev": "tsc-watch --project . --noClear --outDir ./dist --onSuccess \"nodemon ./dist/server.js\" | pino-pretty",
-    "dev:cli": "tsc-watch --project . --noClear --outDir ./dist --onSuccess \"nodemon ./dist/cli/index.js console merge\" | pino-pretty",
+    "dev:cli": "tsc-watch --project . --noClear --outDir ./dist --onSuccess \"nodemon ./dist/cli/index.js workspace user\" | pino-pretty",
     "dev:cli-dry": "tsc-watch --project . --noClear --outDir ./dist --onSuccess \"nodemon ./dist/cli/index.js console merge --dry\" | pino-pretty",
     "dev:debug": "tsc-watch --project . --noClear --outDir ./dist --onSuccess \"nodemon --inspect=0.0.0.0 ./dist/server.js\" | pino-pretty",
```

Comment:
Do we keep this name dev:cli and dev:cli-dry ? Don't we want to have something like dev:cli:console:merge or even just `npm run dev:cli -- console merge` (and remove the "console merge" part hardcoded here) ?
Just a proposition

Reply:
going to check this

Reply:
Well this is not possible since the cli is launched from tsc-watch which launches nodemon which is piped into pino-pretty...